lib/rules/no-return-await.js

```diff
@@ -44,7 +46,19 @@ module.exports = {
             context.report({
                 node: context.getSourceCode().getFirstToken(node),
                 loc: node.loc,
-                messageId: "redundantUseOfAwait"
+                messageId: "redundantUseOfAwait",
+                suggest: [
+                    {
+                        messageId: "removeAwait",
+                        data: { type: "block" },
+                        fix(fixer) {
+                            const range = [node.range[0], node.range[0] + 6];
```

Comment:
This could remove a non-whitespace character right after the `await`, for example:

```js
async () => {
    return await(foo());
}

```

after the suggestions is applied:

```js
async () => {
    return foo()); // Parsing error: Unexpected token )
}

```

Reply:
Also, in a [ReturnStatement](https://tc39.es/ecma262/#sec-return-statement), there should be no linebreaks between the `return` keyword and the expression.

If the code looks like this:

```js
async () => {
    return await // comment
        foo();
}
```

after the suggestion is applied, the fixed code would be:

```js
async () => {
    return // comment
        foo();
}
```

this is not `return foo();`, but the same as `return; foo();`

Reply:
Good catches, thanks! I'll fix these most likely tomorrow.

Reply:
I've fixed the whitespace problem by only removing a space if that character is a space. This might not remove e.g. another whitespace character it could remove, but is the safe choice as not removing the character after await should be fine in all cases.

Reply:
Concerning the case of removing await in a multiline await statement, I've added a check to only add this suggestion if await and its argument both start on the same line.